toolbar.py

```diff
@@ -56,6 +57,18 @@ def __init__(self, pc, toolbar_box):
 
         self._abiword_canvas = pc.abiword_canvas
 
+        # initialize some of the extra stock icons we need
+        factory = Gtk.IconFactory()
+        icons_dir = os.path.join(os.path.abspath(os.path.dirname(__file__)),
+                'icons')
+        factory.add('format-text-super', Gtk.IconSet(
+            GdkPixbuf.Pixbuf.new_from_file(os.path.join(icons_dir,
+                'format-text-super.svg'))))
+        factory.add('format-text-sub', Gtk.IconSet(
+            GdkPixbuf.Pixbuf.new_from_file(os.path.join(icons_dir,
+                'format-text-sub.svg'))))
+        factory.add_default()
```

Comment:
This should be automatic is if they are in the icons directory.  Is it not working automatically?  How does paint activity do this?